services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/ObjectStoreAccess.java

```diff
@@ -120,7 +121,7 @@ public void deleteObjectsWithPrefix(String prefix) {
       headers.put(HeaderKey.AMZ_ACL, "public-read");
     }
 
-    if (!StringUtils.isEmpty(file.getChecksum())) {
+    if (!ObjectUtils.isEmpty(file.getChecksum())) {
```

Comment:
doesn't this change the logik? String.isEmpty() doesn't have the same meaning as Object.isEmpty() does it?

Reply:
According to this documentation, [StringUtils.isEmpty()](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html#isEmpty-java.lang.Object- ), this change should have the same behaviour.